src/main/java/com/easypost/model/TimeInTransit.java

```diff
@@ -152,7 +152,9 @@ public void setPercentile99(final Integer percentile99) {
      * @param percentile the percentile to find the corresponding accuracy for
      * @return the delivery accuracy of the specified percentile
      * @throws EasyPostException when the percentile is not valid
+     * @deprecated Use {@link #getBySmartrateAccuracy(SmartrateAccuracy)} instead.
      */
+    @Deprecated
     public int getSmartRateAccuracy(final String percentile) throws EasyPostException {
```

Comment:
I believe this function can be removed unless a user is using it for some reason

Reply:
I'll go through in a second and remove any function that hasn't been released publicly yet.

Welcome to Java, where everything is deprecated, but nothing is removed.